software/externals/cmake/externals.cmake

```diff
@@ -142,7 +142,7 @@ set(flycapture_revision be27acc7effd83b0cb5648742b915eb4a6181f49)
 set(flycapture_depends)
 set(flycapture_external_args
   ${download_only_args}
-  SOURCE_DIR ${source_prefix}/../drivers/flycapture
+  SOURCE_DIR ${source_prefix}/../atlas-collection/flycapture
```

Comment:
From what I believe there might be the possibility we'll use it again on Valkyrie - unless the wiring/set up is different and NASA is providing pre-installed ROS drivers to use for the beginning. I'm happy to move it back to drivers (along with ptgrey) and convert that to cmake as well. As said, I didn't intend to ignite a too fundamental conversion from one build system to another. Shall we tackle this restructure of drivers now as well or postpone it for another PR/when the robot arrives and we decide to re-use the flycapture/ptgrey elements as far as possible/deemed necessary? What are your thoughts @mauricefallon?